pandas/io/gbq.py

```diff
@@ -553,11 +549,12 @@ def _parse_data(schema, rows):
 def _parse_entry(field_value, field_type):
     if field_value is None or field_value == 'null':
         return None
-    if field_type == 'INTEGER' or field_type == 'FLOAT':
+    if field_type == 'INTEGER':
+        return int(field_value)
+    elif field_type == 'FLOAT':
         return float(field_value)
     elif field_type == 'TIMESTAMP':
-        timestamp = datetime.utcfromtimestamp(float(field_value))
-        return np.datetime64(timestamp)
```

Comment:
1) doesn't really matter, but ok
2) sure, for a single value. you are much better off leaving these as floats, then coercing all in one go if you care about perf (using ``to_datetime``)